apps/prairielearn/elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -156,6 +162,7 @@ def prepare(element_html: str, data: pl.QuestionData) -> None:
         "format",
         "code-language",
         "allow-blank",
+        "block-orientation",
     ]
```

Comment:
When this feature existed previously it was done with `inline=true/false`, but I actually don't like that much.

I'm not sure `block-orientation` is the right thing to call it either though. It doesn't actually change the orientation of the blocks per se, but the way that they are layed out with respect to one another. What would people think about `block-flow=vertical/horizontal`. @mwest1066 @nwalters512 @jonatanschroeder @armandofox  do you have thoughts about what this attribute should be called?

Reply:
I actually prefer inline instead of vertical/horizontal. This should allow future improvements where wrapping is allowed (instead of scrolling) for long content.

Reply:
I think I agree with @jonatanschroeder as well. Id initially had it as a boolean attribute but switched to try and be consistent with selection-placement being a string attribute.

Reply:
You can go ahead and use `inline=true/false` then. I don't have strong feelings about it.

Reply:
Sounds good! I'll go with that.